.gitlab-ci.yml

```diff
@@ -112,6 +112,136 @@ build:mender-client-acceptance-testing:
     paths:
       - checksums
 
+.template:build:mender-dist-packages-image:
+  stage: build
+  needs: []
+  image: docker:git
+  tags:
+    - mender-qa-worker-generic
+  services:
+    - docker:dind
+  before_script:
+    - apk --update --no-cache add bash curl aws-cli
+    - eval "$(curl https://raw.githubusercontent.com/mendersoftware/mendertesting/master/mender-ci-common.sh)"
+  script:
+    # GPG key is available only in protected branches
+    - if [ "$GPG_PRIV_KEY_BUILD" != "" ]; then
+    -   GPG_KEY_BUILD="$GPG_PRIV_KEY_BUILD"
+        ./mender-dist-packages-building/docker-build-image $DISTRO $RELEASE $ARCH
+    - else
+    -   ./mender-dist-packages-building/docker-build-image $DISTRO $RELEASE $ARCH
+    - fi
```

Comment:
this if is redundant, right? empty GPG_PRIV_KEY_BUILD means empty GPG_KEY_BUILD which will be so whether we set it or not. (the difference will be that the var will be defined, but empty, but we check it with -n so should be ok.

Reply:
You are right here, I don't know why we had this safeguard in the original script. I'll simplify it :+1: